### PR TITLE
Alignment with Figma characteristics

### PR DESCRIPTION
**Pull request by libertymayc**
Sync up with Figma - update or add any missing tokens

### CODE REVIEW

File: .changeset/curly-cows-buy.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Changes to Measured characteristic values; change typography -weight token to -fontWeight
```
Comment (joshwooding) [inline .changeset/curly-cows-buy.md:6]: It might be better to have two small PRs than one big PR for these types of changes?

Comment (libertymayc) [reply .changeset/curly-cows-buy.md:6]: good idea thats probably best, will split :)

File: .changeset/curly-cows-buy.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Alignment with Figma characteristics
```
Comment (joshwooding) [inline .changeset/curly-cows-buy.md:6]: Could we go into a bit more detail here about the the changes please

Comment (libertymayc) [reply .changeset/curly-cows-buy.md:6]: done

File: .changeset/curly-cows-buy.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+--uitk-actionable-cursor-hover
+--uitk-actionable-textTransform
+--uitk-draggable-horizontal-cursor-active
+--uitk-navigable-indicator-disabled
+--uitk-ratable-cursor-disabled
+--uitk-color-grey-90-fade-foreground
+--uitk-palette-navigate-indicator-disabled
+
+Removed:
+--uitk-actionable-primary/cta/secondary-fontWeight-hover
```
Comment (joshwooding) [inline .changeset/curly-cows-buy.md:20]: I think it would be clearer to split these out

File: .changeset/curly-cows-buy.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Alignment with Figma characteristics
+
+Added:
+--uitk-accent-borderColor
```
Comment (joshwooding) [inline .changeset/curly-cows-buy.md:9]: Nit: was only moved

File: .changeset/curly-cows-buy.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+--uitk-accent-fontWeight
+--uitk-actionable-cursor-hover
```
Comment (joshwooding) [inline .changeset/curly-cows-buy.md:11]: Nit: was only removed

File: .changeset/curly-cows-buy.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+--uitk-actionable-textTransform
```
Comment (joshwooding) [inline .changeset/curly-cows-buy.md:12]: Nit: was only moved

File: .changeset/curly-cows-buy.md (hunk 4 above)
Comment (libertymayc) [reply .changeset/curly-cows-buy.md]: 🤦‍♀️

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
There seems to be a regression with Switch @ZarkoPernar can you take a look

File: packages/theme/css/foundations/fade.css [hunk 7]
```diff
@@ -8,7 +8,8 @@
   --uitk-color-grey-60-fade-foreground: rgba(197, 201, 208, var(--uitk-palette-opacity-foreground));
   --uitk-color-grey-70-fade-foreground: rgba(180, 183, 190, var(--uitk-palette-opacity-foreground));
   --uitk-color-grey-200-fade-foreground: rgba(97, 101, 110, var(--uitk-palette-opacity-foreground));
-  --uitk-color-grey-800-fade-foreground: rgba(36, 37, 38, var(--uitk-palette-opacity-foreground));
+  --uitk-color-grey-800-fade-foreground: rgba(132, 135, 142, var(--uitk-palette-opacity-foreground));
```
Comment (ZarkoPernar) [inline packages/theme/css/foundations/fade.css:11]: @libertymayc This is the source of the regression. I tracked down the origin of this value and it seems to be for grey-90 not grey-800.

File: packages/theme/css/foundations/fade.css [hunk 8]
```diff
@@ -8,7 +8,8 @@
...
+  --uitk-color-grey-90-fade-foreground: rgba(22, 22, 22, var(--uitk-palette-opacity-foreground));
```
Comment (ZarkoPernar) [inline packages/theme/css/foundations/fade.css:12]: Is this a correct change as well? Seems the same value as below 900

**Comment by ZarkoPernar** [comment]
@joshwooding looks to be some typos in the fade palette variables. Just had a quick chat with Zhihao about potentially compiling these from postcss variables during a build step, since the issue here is the technical limitation of css variables. What do you think?

**Comment by joshwooding** [comment]
> @joshwooding looks to be some typos in the fade palette variables. Just had a quick chat with Zhihao about potentially compiling these from postcss variables during a build step, since the issue here is the technical limitation of css variables. What do you think?

I think it's possible but, I would be cautious about introducing postcss because of that. We should discuss it in a team meeting

**Review by joshwooding — APPROVED** [review]

**Pull request merged**